pipelines/folioRancher/folioDevTools/moduleDeployment/deployModuleFromFeatureBranchEureka/Jenkinsfile

```diff
@@ -0,0 +1,199 @@
+package folioRancher.folioDevTools.moduleDeployment.deployModuleFromFeatureBranchEureka
+
+import org.folio.Constants
+import org.folio.models.RancherNamespace
+import org.folio.utilities.Logger
+import org.folio.utilities.GitHubClient
+import org.folio.models.EurekaNamespace
+import org.folio.models.FolioModule
+import org.folio.models.EurekaRequestParams
+import org.folio.rest_v2.eureka.Eureka
+
+// TODO: RANCHER-1700 - get rid off custom branch mention once feature development is done
+@Library('pipelines-shared-library@RANCHER-1700') _
+
+/** Job properties and parameters */
+properties([
+  buildDiscarder(logRotator(numToKeepStr: '30')),
+  disableConcurrentBuilds(),
+  parameters([
+    folioParameters.moduleName(),
+    folioParameters.branchWithRef('MODULE_BRANCH', 'MODULE_NAME'),
+    string(name: 'MAVEN_ARGS', defaultValue: '-DskipTests', description: 'Maven build arguments'),
+    string(name: 'TENANTS_LIST', defaultValue: 'diku', description: 'Comma separated list of Tenant Names to deploy module to'),
+    folioParameters.cluster(),
+    folioParameters.namespace(),
+    folioParameters.configType(),
+    folioParameters.agent(),
+    folioParameters.refreshParameters(),
+  ])
+])
+
+if (params.REFRESH_PARAMETERS) {
+  currentBuild.result = 'ABORTED'
+  return
+}
+
+
+/** Job variables */
+Logger logger = new Logger(this, env.JOB_BASE_NAME)
+
+/** Collect Rancher Namespace Configuration */
+EurekaNamespace namespace = new EurekaNamespace(params.CLUSTER, params.NAMESPACE)
+  .withDeploymentConfigType(params.CONFIG_TYPE) as EurekaNamespace
+
+/** Assign Desired Resource Profile for Environment */
+namespace.addDeploymentConfig(folioTools.getPipelineBranch())
+
+/** Assign Default Tenant Id to Namespace instance */
+String defaultTenantId = 'diku'
+namespace.withDefaultTenant(defaultTenantId)
+
+/** Collect Folio module configuration */
+FolioModule module = new FolioModule()
+module.name = params.MODULE_NAME
+
+/** Collect feature branch and hash */
+String featureBranch = params.MODULE_BRANCH
+String featureHash = new GitHubClient(this).getBranchInfo(module.getName(), featureBranch).commit.sha.take(7)
+
+/** Collect Maven arguments */
+String mavenArguments = params.MAVEN_ARGS.trim()
+
+/** Init Eureka Application Instance */
+Eureka eureka = new Eureka(this, namespace.generateDomain('kong'), namespace.generateDomain('keycloak'))
+
+Boolean skipStage = true
+
+
+/** Pipeline */
+ansiColor('xterm') {
+  node(params.AGENT) {
+    try {
+      stage('Ini') {
+        buildName "#${params.MODULE_NAME}.${env.BUILD_ID}"
+        buildDescription "Env: ${namespace.getWorkspaceName()}\nBranch: ${params.MODULE_BRANCH}\n" + "Config: ${params.CONFIG_TYPE}\n${currentBuild.getBuildCauses()[0].shortDescription}"
+      }
+
+      stage('[Git] Checkout module source') {
+        if(skipStage) {
```

Comment:
Is this is for testing purposes? And if yes, please remove

Reply:
@OHaimanov 
Yep you're right its just to speed up testing process.
Once evrything works and looks good these lines will be deleted.